openwisp_users/tests/utils.py

```diff
@@ -57,7 +50,12 @@ def _create_operator(self, organizations=[], **kwargs):
         )
         opts.update(kwargs)
         operator = User.objects.create_user(**opts)
-        operator.user_permissions.add(*self.get_operator_permissions())
+        groups = Group.objects.filter(name__in=['Administrator', 'Operator'])
+        operator.groups.set(groups)
+        permissions = Permission.objects.filter(
+            Q(codename__endswith='book') | Q(codename__endswith='shelf')
+        )
```

Comment:
I see, it seems we need to add additional permissions to the operator, but wouldn't it be better that 'shelf' and 'book' come from an optional class variable? 

Okay, I see that Fed has asked in the issue to remove `operator_permission_filter` but hardcoding it might not be the best thing because if a module needs to add more permissions or remove these permissions, this wouldn't be ideal. What do you think @nemesisdesign ?

Reply:
The issue required that I should assign both Administrator and Operator groups.